### ISSUE TITLE
Proper packaging for linux [Flatpak/AppImage]

### ISSUE THREAD

**Issue by Twig6943** [BrisklyDev/brisk#67]
title.

**Comment by AminBhst** [comment]
I don't know if I can support AppImage since now brisk is shipped with an auto updater module and there is no way to make that work with AppImage. I'll see what I can do about flatpack.

**Comment by DaRandomCube** [comment]
Flatpak also doesn't support auto updater, but it's the better option due to better desktop integration

**Comment by AminBhst** [comment]
@DaRandomCube Yeah but at least with Flatpak you actually have the option to update the app yourself. I'm not a fan of AppImage because there is no way to make updating it a smoother process.

**Comment by AminBhst** [comment]
Available on [Flathub](https://flathub.org/apps/io.github.BrisklyDev.Brisk)

**Comment by johndo100** [comment]
> I don't know if I can support AppImage since now brisk is shipped with an auto updater module and there is no way to make that work with AppImage. I'll see what I can do about flatpak.

AppImage can be updated via third-party tools like Gear Lever; you just need to release the proper files in the GitHub repository. See the documentation: https://gearlever.mijorus.it/docs/github-updates

**Comment by Twig6943** [comment]
> > I don't know if I can support AppImage since now brisk is shipped with an auto updater module and there is no way to make that work with AppImage. I'll see what I can do about flatpak.
> 
> AppImage can be updated via third-party tools like Gear Lever; you just need to release the proper files in the GitHub repository. See the documentation: https://gearlever.mijorus.it/docs/github-updates

AppImages have package managers such as soar and am as well
